stable/atlassian-confluence/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Could you please specify the owners of the chart in this file?

Reply:
Sorry, this is my first pr :-) and tell me whether I need to put my name `int128` or your name `mattjmcnaughton` in the owners.

Reply:
Got it. I added `maintainers`.